### PR TITLE
update for dqm live configuraiton as in pr12084 pr12410

### PR DESCRIPTION
**Pull request by sushilchauhan**
updated the live configuration for online BS for dqm with changes as in PR#12084, PR#12410 and few other minor changes. This will be used for 2016 running.

### CODE REVIEW

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
+1

File: DQM/BeamMonitor/plugins/BeamMonitor.cc [hunk 1]
```diff
@@ -54,7 +54,7 @@ void BeamMonitor::formatFitTime(char *ts, const time_t & t )  {
              
   if (year < 1995){                                                                   
         edm::LogError("BadTimeStamp") << "year reported is " << year <<" !!"<<std::endl;
-        year = 2015; //overwritten later by BeamFitter.cc for fits but needed here for TH1
+        year = 2016; //overwritten later by BeamFitter.cc for fits but needed here for TH1
```
Comment (davidlange6) [inline DQM/BeamMonitor/plugins/BeamMonitor.cc]: could you instead retrieve the current year from ctime.h?

**Comment by davidlange6** [comment]
@sushilchauhan - and aside from my comment above, please change the name of the PR to be more descriptive of its contents. Thanks!

**Comment by davidlange6** [comment]
ping @sushilchauhan @deguio

**Comment by sushilchauhan** [comment]
my apologies David for late reply. I will try to fix it by tomorrow. I hope this okay with you.

**Comment by davidlange6** [comment]
@sushilchauhan - any news - you just need to click "edit" at the top of the page

**Comment by sushilchauhan** [comment]
just created a new PR#14316 with some additional changes and the ones requested here. My sincere apologies for delay and inconvenience caused by this.

**Comment by dmitrijus** [comment]
please close this.

**Pull request closed without merging**